content/solana-nft.md

```diff
@@ -0,0 +1,567 @@
+ 1. Uploads the asset files to the specified storage type and creates a `.cache` file
+ 2. Creates an on-chain account referred to as a Candy Machine that temporarily stores the links to the uploaded collection and correspond to the `.cache` file. The on-chain Candy Machine account is then used to distribute the collection once minting goes live.
+
+After running the `upload` command, it is best practice to also run the `verify_upload` command. This command will check that each entry in the `.cache` file matches the on-chain Candy Machine account. If any of the URI entries do not match the information stored on on-chain, you will need to re-run the upload command.
+
+Once the upload has been successfully verified, the Candy Machine is ready to mint tokens. The Candy Machine configuration settings will how tokens are minted. Minting is often done through a frontend UI and separated between whitelist and public minting.
+
+Note that the data stored on the Candy Machine account is only used to manange the distribution of the collection. Once a Candy Machine is fully minted, the data stored is no longer useful and the rent for the account can be fully recovered using the `withdraw` command. The `withdraw` command must be executed with the `keypair` that created the Candy Machine.
+
+The final step in creating an NFT collection is for the creator to sign the NFTs and verify themselves as the creator. It is important for a creator to sign the collection to prevent fraudulent collections. Since the creator field of a collection can specify any address, signing an NFT proves that the creator specified in the creator field also verified that the NFT was actually created by them.
+
+# Demo
+
+Let’s put all of this into practice by creating a Candy Machine and minting our collection using the Candy Machine UI.
+
+### 1. Download the starter code
+
+Let's begin by downloading the starter [code](https://github.com/ZYJLiu/metaplex-starter). The starter code includes an `assets` folder and the configuration file for our Candy Machine, `config.json`. Additionally, it includes two helper scripts. One creates a new keypair we will use for this lesson. The other creates the whitelist token we will later use to enable whitelist settings.
+
+### 2. Setup
+While in the `metaplex-starter` folder, lets first install the dependencies for our starter code:
+```sh
+npm install
+```
+
+Next let's set up a new keypair to use for this lesson and airdrop some devnet SOL using our helper script. Run the following command:
+
+```sh
+npm start
+```
+You should now have see a new `metaplex-starter.json` file.
+Copy the keypair from the new file and import it into Phantom
+
+![Gif of Phantom Import](../assets/solana-nft-phantom.gif)
+
+### 3. Download Metaplex
+
+While in the `metaplex-starter` directory, clone the Metaplex repo:
+
+```sh
+git clone https://github.com/metaplex-foundation/metaplex.git
+```
+
+If you don't already ts-node installed globally go ahead and do that now:
```

Comment:
Note to self to update this

Reply:
whoops - I shoulda caught that before closing the convo... @jamesrp13 - there's a couple other little fixes:

- line 100 remove double whitespace
- lines 161 & 194
  - devnet --> Devnet
  - mainnet --> Mainnet
- line 198 needs a newline between ending triple-backtick and next sentence start
- line 199 fix: either needs a newline inserted, or the next line appended
- line 212 fix and format, sugg: "If you don't already have `ts-node` and `typescript` installed globally, go ahead and do that now:"